src/rt/sections_win64.d

```diff
@@ -137,12 +136,16 @@ extern(C)
      */
     extern __gshared
     {
+        void* __ImageBase;
+
         void* _deh_beg;
         void* _deh_end;
 
-        int __xc_a;      // &__xc_a just happens to be start of data segment
-        //int _edata;    // &_edata is start of BSS segment
-        //void* _deh_beg;  // &_deh_beg is past end of BSS
+        void* _data_beg;
+        void* _data_end;
+
+        void* _bss_beg;
+        void* _bss_end;
```

Comment:
Removed. They were used t reduce the code to D data only, but that's no longer in this PR.